tests/domain/entities/test_enrolment.py

```diff
@@ -0,0 +1,36 @@
+from app.domain.entities.enrolment_request import EnrolmentFilters, EnrolmentRequest
+from tests.test_data.enrolment_data_provider import EnrolmentDataProvider
+
+test_data = EnrolmentDataProvider()
+
+
+def test_enrolment_init():
+    enrolment = EnrolmentRequest(
```

Comment:
may need to revise EnrolmentRequest structure. student_id, course_id

Reply:
Changed